#get_symbolics
#reference of the action 
uint32 actionID
#reference of the object
uint32 objectID
#reference of the component
uint32 componentID
---
#symbolic ID for the action, the object and the component
uint32[] symbolicID